go/staking/api/api.go

```diff
@@ -642,11 +648,14 @@ type DebondingDelegation struct {
 	DebondEndTime epochtime.EpochTime `json:"debond_end"`
 }
 
-// Genesis is the initial ledger balances at genesis for use in the genesis
-// block and test cases.
+// Genesis is the initial staking state for use in the genesis block.
 type Genesis struct {
 	Parameters ConsensusParameters `json:"params"`
 
+	// TokenSymbol is the token's ticker symbol.
```

Comment:
This grouping looks strange, either move these to their own group or add comments everywhere.

Reply:
I've added comments for all the fields.